.github/workflows/build-toolchain.yml

```diff
@@ -313,6 +375,8 @@ jobs:
       yams_revision: ${{ needs.context.outputs.yams_revision }}
       zlib_revision: ${{ needs.context.outputs.zlib_revision }}
       ANDROID_API_LEVEL: ${{ needs.context.outputs.ANDROID_API_LEVEL }}
+      HOST_CMAKE_C_FLAGS: ${{ needs.context.outputs.WINDOWS_CMAKE_C_FLAGS }}
+      HOST_CMAKE_CXX_FLAGS: ${{ needs.context.outputs.WINDOWS_CMAKE_CXX_FLAGS }}
```

Comment:
What is HOST here? Is it the host or the build? CMake mixes up the terms, and that makes this very confusing. I'd prefer a different name unless this is something that is baked into CMake.